packages/rln/src/contract/rln_base_contract.ts

```diff
@@ -456,12 +458,12 @@ export class RLNBaseContract {
         `Rate limit must be between ${RATE_LIMIT_PARAMS.MIN_RATE} and ${RATE_LIMIT_PARAMS.MAX_RATE}`
       );
     }
-    return this.contract.register(idCommitment, rateLimit, []);
+    return this.contract.register(idCommitmentBigInt, rateLimit, []);
   }
 
-  public async withdraw(token: string, holder: string): Promise<void> {
+  public async withdraw(token: string, from: string): Promise<void> {
```

Comment:
if `from` wallet address? if so, I would name it like this

Reply:
Like? :D

Reply:
like `wallet` ;)

Reply:
done